src/encoded/tests/features/biosamples.feature

```diff
@@ -16,7 +16,7 @@ Feature: Biosamples
         And I wait for the content to load
         When I click the link to "/search/?type=Biosample&organism.scientific_name=Homo+sapiens"
         Then I should see an element with the css selector "div.search-results"
-        And I should see "Showing 25 of 44 results"
+        And I should see "Showing 25 of 49 results"
```

Comment:
I agree, this change doesn't make sense. You need to get to the bottom of the number change. Why it was 44 and became 49 after you adding one biosample?

Reply:
Following the "n+1" plan for now (total of 45 with my 1 added insert), will address the bdd-user permissions issue in another ticket.